test/testdata/simpletests/datavolume.json

```diff
@@ -0,0 +1,8 @@
+{
+  "TaskDefinition": "datavolume",
```

Comment:
Could we provide a description here of what each task is supposed to do in the test definition? This might help diagnosing test failures.